src/Sarif.UnitTests/Writers/ResultLogJsonWriterTests.cs

```diff
@@ -330,95 +288,22 @@ public void ResultLogJsonWriter_CannotWriteInvocationTwice()
-            ""fileLocation"": {
-              ""uri"": ""file:///C:/src/a.cs""
-            },
-            ""region"": {
-              ""startLine"": 3,
-              ""startColumn"": 12
-            }
-          },
-          ""message"": {
-            ""text"": ""This is a test""
-          },
-          ""level"": ""error"",
-          ""timeUtc"": ""2016-04-29T00:00:00.000Z"",
-          ""exception"": {
-            ""kind"": ""System.AggregateException"",
-            ""message"": {
-              ""text"": ""Bad thing""
-            },
-            ""innerExceptions"": [
-              {
-                ""kind"": ""System.ArgumentNullException"",
-                ""message"": {
-                  ""text"": ""x cannot be null""
-                },
-                ""stack"": {
-                  ""frames"": [
-                    {
-                      ""location"": {
-                        ""physicalLocation"": {
-                          ""fileLocation"": {
-                            ""uri"": ""file:///C:/src/a.cs""
-                          },
-                          ""region"": {
-                            ""startLine"": 10
-                          }
-                        },
-                        ""fullyQualifiedLogicalName"": ""N1.N2.C.M1""
-                      },
-                      ""module"": ""a.dll""
-                    },
-                    {
-                      ""location"": {
-                        ""physicalLocation"": {
-                          ""fileLocation"": {
-                            ""uri"": ""file:///C:/src/a.cs""
-                          },
-                          ""region"": {
-                            ""startLine"": 6
-                          }
-                        },
-                        ""fullyQualifiedLogicalName"": ""N1.N2.C.M2""
-                      },
-                      ""module"": ""a.dll""
-                    }
-                  ]
-                }
-              }
-            ]
-          }
-        }";
 
         [Fact]
         public void ResultLogJsonWriter_WritesConfigurationNotifications()
         {
-            string expected =
-@"{
-  ""$schema"": """ + SarifUtilities.SarifSchemaUri + @""",
-  ""version"": """ + SarifUtilities.SemanticVersion + @""",
-  ""runs"": [
-    {
-      ""tool"": {
-        ""name"": ""DefaultTool""
-      },
-      ""columnKind"": ""utf16CodeUnits"",
-      ""invocations"": [
-        {                
-          ""configurationNotifications"": [
-" + SerializedNotification + @"
-          ]
-       }
-     ]
-    }
-  ]
-}";
+            string expected = CreateCurrentV2SarifLogText(
+                resultCount: 0,
+                (log) => {
+                    log.Runs[0].Invocations = new List<Invocation>
+                    {
+                        new Invocation
+                        {
+                            ConfigurationNotifications = new List<Notification>(s_notifications)
```

Comment:
>s_notifications [](start = 80, length = 15)

Hah. Really nice.

Reply:
All the pieces were here. :)

---
In reply to: [256121795](https://github.com/Microsoft/sarif-sdk/pull/1277#discussion_r256121795) [](ancestors = 256121795)